web/templates/draft.hbs

```diff
@@ -16,7 +16,7 @@
                     </div>
                 </span>
                 <span class="recipient_bar_controls"></span>
-                <div class="recipient_row_date" title="{{t 'Last modified'}}">{{ time_stamp }}</div>
+                <div class="recipient_row_date restore-overlay-message tippy-zulip-delayed-tooltip" data-tooltip-template-id="last-modified-tooltip-template">{{time_stamp}}</div>
```

Comment:
Why is `restore-overlay-message` added here?

Reply:
Also, why use a tooltip template for this one? We usually only need those if there's HTML in the template, and this one should be able to just have the string "Last modified".